org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/editor/folding/DefaultFoldingRangeAcceptor.java

```diff
@@ -0,0 +1,61 @@
+/*******************************************************************************
+ * Copyright (c) 2010 Michael Clay and others.
```

Comment:
I think we should use `2021` in the copyright. @cdietrich what do you think?

Reply:
Agriid

Reply:
@Darkgaja Could you fix this?